packaging/etc/engine-config/engine-config.properties

```diff
@@ -550,6 +550,9 @@ LiveSnapshotPerformFreezeInEngine.type=Boolean
 LiveSnapshotFreezeTimeout.description=The live snapshot command without memory timeout in minutes
 LiveSnapshotFreezeTimeout.type=Integer
 LiveSnapshotFreezeTimeout.validValues=1..3000
+VolumeUtilizationChunkInMB.description="Size of extension chunk in megabytes. Use higher values to extend in bigger chunks."
```

Comment:
how about adding here that if this is changed, one is also expected to change the corresponding value in vdsm?

Reply:
AFAIU, the VDSM value will "go away" some day, after using the engine value.
@nirs what do you think worth adding comment. If so, maybe the comment should also mention that this value should replace the VDSM one, thus also need to be removed after VDSM's value is gone?

Reply:
@ahadas @nirs Please check the new version. I tried to be consistent with the existing style :)